README.md

```diff
@@ -28,15 +28,21 @@ The development of TTG was motivated by _irregular_ scientific applications like
 #include <ttg.h>
 
 int main(int argc, char *argv[]) {
+  // initialization
   ttg::initialize(argc, argv);
-
+  // a simple template task
   auto tt = ttg::make_tt([]() { std::cout << "Hello, World!\n"; });
 
+  // make sure the graph is connected (simple for a single node)
   ttg::make_graph_executable(tt);
+  // start executing any available tasks
   ttg::execute();
+  // add a single task into the taskpool
```

Comment:
since we never introduce implementation details like taskpools, I prefer something like `// create task to kickstart computation`